### ISSUE TITLE
Variable names should have a meaning

### ISSUE THREAD

**Issue by OzHache** [harry-chew/Vehicle-Configurator#4]
https://github.com/harry-chew/Vehicle-Configurator/blob/201b342ffa6071750f5a73fb59b7ddd75770dbd0/Assets/Scripts/FreeRotateCamera.cs#L41

lol isn't going to work here. If it is important enough to need a variable, it should be important enough to have a name that says what it is :) Points for the use of lol in your code though.

**Comment by harry-chew** [comment]
Yeah, this doesn't work xD I won't leave it like this once I have got it working, but thank you.
